elastic/kibana
# [Security Solutions] Sync EA transform with risk engine

## Summary

@elastic/security-entity-analytics

When the risk engine is enabled, users must wait at least one hour to see risk score data.
This PR fixes the issue by scheduling the latest transform after the risk score task finishes.

The first time the risk score runs, it will call `start transform`. That is necessary to ensure the transform doesn't run on an empty index. On the subsequent runs, it will call `schedule transform now`. 

How to test it?
* Open kibana with an empty ES
* Make sure you have the appropriate license
* Ingest some events and generate alerts
* Enable the risk engine
* Open the Entity Analytics Dashboard 
* You should see risk score data (if it isn't there yet, refresh the page)

### Checklist

Delete any items that are not applicable to this PR.
